modules/guides/pages/select.adoc

```diff
@@ -56,7 +56,7 @@ The result includes each row found.
 
 [source,csharp]
 ----
-include::dotnet-sdk:hello-world:example$StartUsing.cs[tag=n1ql-query,indent=0]
+include::dotnet-sdk:hello-world:example$StartUsing.csx[tag=n1ql-query,indent=0]
```

Comment:
Correct to change for now,
but .csx seems to have been the wrong direction,
and it will eventually change back.